lib/main.dart

```diff
@@ -42,11 +43,8 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp.router(
-      theme: ThemeData(
-        primarySwatch: Colors.blue,
-        brightness: Brightness.light,
-        fontFamily: Assets.fonts.neuzeit,
-      ),
+      theme:
+          AppTheme.light.copyWith(scaffoldBackgroundColor: Colors.transparent),
```

Comment:
I think we can set `the scaffoldBackgroundColor` in `app_theme` instead of copy the value here?

Reply:
@Wadeewee I will remove this line once we use the real screens. This was for the PoW to show that the theme works because the texts are white. 😄

Reply:
OK Noted 🚀